pipeline-model-api/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/ast/InvisibleGlobalWhenCondition.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2021, CloudBees, Inc.
```

Comment:
already 2021? nice.

Reply:
I’m being proactive since I don’t expect this to merge this year. =)